build-tools/packages/build-cli/docs/generate.md

```diff
@@ -312,6 +313,39 @@ DESCRIPTION
 
 _See code: [src/commands/generate/packlist.ts](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/generate/packlist.ts)_
 
+## `flub generate releaseNotes`
+
```

Comment:
Should there be a description of what it does / is used for here?

Reply:
Yes, I'll add it.